src/containers/Configure/Configure.js

```diff
@@ -52,102 +52,104 @@ const Configure = ({ userQuote, allUserQuotes }) => {
 
   return (
     <>
-      {formCompleted && <Redirect push to="/historical" />}
+      {formCompleted && <Redirect to="/historical"/>}
```

Comment:
If the url history doesn't work when using the back button, we can add the `push` attribute inside the Redirect and that should fix it

Reply:
Roger

Reply:
the push idea works in this instance